code/lib/builder-manager/src/utils/files.test.ts

```diff
@@ -2,43 +2,28 @@ import { platform } from 'os';
 import { sanitizePath } from './files';
 
 const os = platform();
+const isWindows = os === 'win32';
 
-if (os !== 'win32') {
-  test('sanitizePath', () => {
-    const addonsDir = '/Users/username/Projects/projectname/storybook';
-    const text = 'demo text';
-    const file = {
-      path: '/Users/username/Projects/projectname/storybook/node_modules/@storybook/addon-x+y/dist/manager.mjs',
-      contents: Uint8Array.from(Array.from(text).map((letter) => letter.charCodeAt(0))),
-      text,
-    };
-    const { location, url } = sanitizePath(file, addonsDir);
+test('sanitizePath', () => {
+  const addonsDir = isWindows
+    ? 'C:\\Users\\username\\Projects\\projectname\\storybook'
+    : '/Users/username/Projects/projectname/storybook';
+  const text = 'demo text';
+  const file = {
+    path: isWindows
+      ? 'C:\\Users\\username\\Projects\\projectname\\storybook\\node_modules\\@storybook\\addon-x+y\\dist\\manager.mjs'
+      : '/Users/username/Projects/projectname/storybook/node_modules/@storybook/addon-x+y/dist/manager.mjs',
+    contents: Uint8Array.from(Array.from(text).map((letter) => letter.charCodeAt(0))),
+    text,
+  };
+  const { location, url } = sanitizePath(file, addonsDir);
 
-    expect(location).toMatchInlineSnapshot(
-      `"/Users/username/Projects/projectname/storybook/node_modules/@storybook/addon-x+y/dist/manager.mjs"`
-    );
-    expect(url).toMatchInlineSnapshot(
-      `"./sb-addons/node_modules/%40storybook/addon-x%2By/dist/manager.mjs"`
-    );
-  });
-}
-
-if (os === 'win32') {
-  test('sanitizePath - 1', () => {
-    const addonsDir = 'C:\\Users\\username\\Projects\\projectname\\storybook';
-    const text = 'demo text';
-    const file = {
-      path: 'C:\\Users\\username\\Projects\\projectname\\storybook\\node_modules\\@storybook\\addon-x+y\\dist\\manager.mjs',
-      contents: Uint8Array.from(Array.from(text).map((letter) => letter.charCodeAt(0))),
-      text,
-    };
-    const { location, url } = sanitizePath(file, addonsDir);
-
-    expect(location).toMatchInlineSnapshot(
-      `"C:\\\\Users\\\\username\\\\Projects\\\\projectname\\\\storybook\\\\node_modules\\\\@storybook\\\\addon-x+y\\\\dist\\\\manager.mjs"`
-    );
-    expect(url).toMatchInlineSnapshot(
-      `"./sb-addons/node_modules/%40storybook/addon-x%2By/dist/manager.mjs"`
-    );
-  });
-}
+  expect(location).toMatchInlineSnapshot(
+    isWindows
```

Comment:
I wonder if this would be better as "toEqual" rather than inlineSnapshot, as jest normally overwrites inlineSnapshots if accepted, and therefore the ternary might go away

Reply:
Or maybe:

```ts
if (isWindows) {
  expect(location).toMatchInlineSnapshot(`"C:\\\\Users\\\\username\\\\Projects\\\\projectname\\\\storybook\\\\node_modules\\\\@storybook\\\\addon-x+y\\\\dist\\\\manager.mjs"`);
} else {
  expect(location).toMatchInlineSnapshot(`"/Users/username/Projects/projectname/storybook/node_modules/@storybook/addon-x+y/dist/manager.mjs"`);
}
```

Reply:
@kasperpeulen  -> expect shouldn't be called conditionally. Of course I can just disable the check for this line [eslint[jest/no-conditional-expect](https://github.com/jest-community/eslint-plugin-jest/blob/v24.7.0/docs/rules/no-conditional-expect.md)]

Reply:
I see, to equal may be better than yes